tensordict/prototype/tensorclass.py

```diff
@@ -590,13 +578,7 @@ def _memmap_like(
     out_td = self._tensordict.memmap_like(prefix=prefix)
     out_other = {}
     for key, val in self._non_tensordict.items():
-        if is_tensorclass(val):
-            if prefix is not None:
-                out_other[key] = val.memmap_like(prefix=prefix / key)
-            else:
-                out_other[key] = val.memmap_like()
-        else:
-            out_other[key] = val
+        out_other[key] = val
```

Comment:
similarly here, we can just do `out_other = dict(self._non_tensordict)` for a shallow copy